tests/unit/lang/translator/de.txt

```diff
@@ -0,0 +1,162 @@
+Kein Filamentsensor erkannt. Überprüfen Sie, ob der Sensor angeschlossen ist, und versuchen Sie es erneut.
+Kein USB
+Jetzt kalibrieren\nwir die Entfernung\nzwischen der Spitze\nder Düse und\ndas Druckblech.
+Düse
+Düse ist zu kalt.
+Aus
+An
+Bei Neustart
+Einmal
+Sobald der Drucker\nbeginnt, Kunststoff\nzu extrudieren,\nstellen Sie die Höhe\nder Düse durch Knopf\nDrehen ein, bis der\nFaden am Druckblech\nhaftet.
+Parke
+Legen Sie während der Kalibrierung der ersten 4 Punkte ein Blatt Papier unter die Düse. Wenn die Düse das Papier berührt, schalten Sie den Drucker sofort aus!
+Bitte reinigen Sie die Düse zur Kalibrierung. Klicken Sie auf WEITER, wenn Sie fertig sind.
+Bitte schließen Sie ein USB-Laufwerk an und versuchen Sie es erneut.
+Bitte schließen Sie\nden USB-Stick an,\nder mit Ihrem MINI \ngeliefert wurde und\nDrucker zurücksetzen\nzum Firmwareflashen
+Bitte legen Sie Stahlblech auf das Heizbett.
+Bitte entfernen Sie Stahlblech vom Heizbett.
+Bitte warten
+VORHEIZEN
+Vorheizen
+VORHEIZEN LADEN
+VORHEIZEN ENTLADEN
+Bereite Rammen vor
+WEITER drücken und\nFilament in den\nExtruder schieben.
+Drücken Sie WEITER,\num den Selbsttest\nauszuführen, bei dem\nnach potenziellen\nProblemen mit der\nMontage gesucht\nwird.
+Drucken
+Drucklüfter
+Druck wird enden
+Druckzeit
+FILAMENT REINIGEN
+Filament reinigen
+Reinige
+QR Test
+Ramme
+NEU HEIZEN
+Restzeit
+RESET DRUCKER
+WIEDERHOLEN
+Crash Dump speichern
+Einstellungen speichern
+Filamenttyp wählen
+Aktivieren, wenn Sie\nautomatisch aktua-\nlisierte Firmware\nvon einem USB-STICK\nflashen möchten.
+SelfTest
+SELBSTTEST
+Sende Info mit QR
+Service
+Einstellungen
+Einstellungen erfolgreich geladen
+Leise
+Ton Modus
+Ton-Typ
+Geschwindigkeit
+StandardAlert
+StandardPrompt
+Statische IPv4-Adressen wurden nicht festgelegt.
+Statistik
+STOPP
+Support
+System Info
+TEMPERATUR
+Temperatur
+Test
+Die Kalibrierung der ersten Schicht konnte nicht abgeschlossen werden. Überprüfen Sie die Verkabelung des Druckers, die Düse und die Achsen und starten Sie die Kalibrierung erneut.
+Der Drucker ist nicht kalibriert. Erste-Schicht-Kalibrierung starten?
+Der Selbsttest wurde\nnicht beendet.\nÜberprüfen Sie die\nVerkabelung und die\nAchsen des Druckers\ndoppelt. Starten Sie\ndann den Selbsttest\nerneut.
+Die Einstellungen wurden erfolgreich in der Datei \lan_settings.ini\ gespeichert.
+Die Statusleiste ist\nam unteren Rand des\nBildschirms. Sie\nenthält Infos über:\n - Düsentemperatur    \n - Heizbetttemp.   \n - Drucktempo\n - Höhe der Z-Achse   \n - Filamenttyp
+Die XYZ-Kalibrierung konnte nicht abgeschlossen werden. Überprüfen Sie die Verkabelung und die Achsen des Druckers doppelt und starten Sie dann die XYZ-Kalibrierung erneut.
+Es gab einen Fehler beim Speichern der Einstellungen in der Datei \lan_settings.ini\.
+Diese Operation kann nicht rückgängig gemacht werden, die aktuelle Konfiguration geht verloren! Sind Sie wirklich sicher, dass Sie den Drucker auf die Werkseinstellungen zurücksetzen wollen?
+Timeout
```

Comment:
not translated?

Reply:
These are raw data for the unit test. I'm still waiting for another batch of translated strings to finalize the UI/UX